rifle.py

```diff
@@ -36,17 +40,49 @@ def __init__(self, universe, DRAWING_SCALE):
         self.word_length_min_ = 3
         self.word_length_range = 3
 
-
+    def getTargetID(self,input): 
```

Comment:
In PEP8, methods are also with underscores. This should be 
`def get_target_id(self, input):`